src/formatter.ts

```diff
@@ -42,16 +41,16 @@ function formatResults(
       return;
     }
 
-    const areAllMessagesTodo =
-      counts.errorCount === 0 &&
-      counts.warningCount === 0 &&
-      counts.todoCount > 0;
+    const areAllMessagesTodo = messages.every(
+      (message) => message.severity === -1
```

Comment:
It'd be great to have a const to represent the `-1` severity (obviously not on this line).

```
const TODO_SEVERITY = -1;
```

Reply:
Good idea. Done. I created a const file that export this to other places too.